### DISCUSSION TITLE
建议增加截图功能的复制到剪贴板选项

### DISCUSSION THREAD

**Discussion by XiaoLinXiaoZhu** [the1812/Bilibili-Evolved#4806] [功能建议 / Ideas]
**问题描述与建议理由**

尊敬的开发团队，

我一直在使用你们的项目，并对它的功能和性能感到非常满意。不过，在使用截图功能时，发现仅能保存或删除截图，这在需要快速分享或编辑截图至如QQ、微信、飞书等通讯软件时，显得有些不便。当然也可以右键之后点击复制图片，但是那样的话需要等待右键选项打开，并且在多个选项中点击复制图像，打断了工作流的连贯性。

![image](https://github.com/the1812/Bilibili-Evolved/assets/102965763/b453ca3c-0b87-417d-a086-9890f196aeac)

---

**期望解决方案**

我建议在截图功能中增加一个“复制到剪贴板”的按钮，或者增加一个点击截图复制到剪贴板的功能。通过点击该按钮或者直接点击截图，截图将被自动复制到系统的剪贴板中，用户可以立即在其他应用程序中进行粘贴。

---

**其他信息**

![image](https://github.com/the1812/Bilibili-Evolved/assets/102965763/ef4dbaa0-3531-42cd-8771-ddb1a60b36ce)
比如说此处，增加点击非按钮区域可以直接复制到剪贴板将会十分有效

---

**结语**

我深信这一小改动将极大提升用户体验，为更多用户带来便利。感谢你们的努力和贡献，期待看到这一改进在未来版本中实现。
谢谢！

**Comment by Geeyun-JY3** [comment]
先前相关讨论关联：#2409、#3214

**Comment by the1812** [comment]
以前没加是因为 Firefox 不支持, 现在 Firefox 127 支持了, 可以考虑下了
